Automattic/jetpack
# Scheduled Updates: Add Scheduled Updates submenu item

Fixes https://github.com/Automattic/jetpack/issues/36921

## Proposed changes:

* Moving Scheduled Updates submenu item from wpcomsh to wpcom-site-menu.php instead.
* This PR needs to be tested with https://github.com/Automattic/wpcomsh/pull/1822 to make sure there's no duplication.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* You can use Jetpack Beta tester and apply this PR or jetpack rsync this to your testing WoA developer site.
* Make sure `Scheduled Updates` submenu item is showing fine under Plugins.
![Screen Shot 2024-04-25 at 5 19 03 PM](https://github.com/Automattic/jetpack/assets/4074459/d87a7935-21d9-46a6-9817-a38b23d4857a)
